src/index.ts

```diff
@@ -52,12 +53,10 @@ proxy.on('error', err => console.error(err));
 // Locate and read `package.nw`
 const [ data, stat ] = await async function() {
 	const path = argv.package ?? function() {
-		switch (process.platform) {
-			case 'darwin': return new URL('./Library/Application Support/Steam/steamapps/common/Screeps/package.nw', `${pathToFileURL(os.homedir())}/`);
-			case 'win32': return 'C:\\Program Files (x86)\\Steam\\steamapps\\common\\Screeps\\package.nw';
-			default: return undefined;
-		}
-	}();
+    const steam = getGamePath(464350);
+      if (!steam || !steam.game) return undefined;
+      return steam.game.path + "\\package.nw";
```

Comment:
This looks wrong. Mac (and Linux) uses forward slash. Let's just check `process.platform`?